### PR TITLE
#423 fix DateTimeOffset Ruby rendering

### PR DESCRIPTION
**Pull request by mhornbacher**
This resolves 2 issues as described in the PR

### CODE REVIEW

File: src/DotLiquid.Tests/Util/StrFTimeTests.cs [hunk 1]
```diff
@@ -39,12 +38,30 @@ public class StrFTimeTests
         [TestCase("%y", ExpectedResult = "12")]
         [TestCase("%Y", ExpectedResult = "2012")]
         [TestCase("%", ExpectedResult = "%")]
-        public string TestFormat(string format)
-        {
-            using (CultureHelper.SetCulture("en-GB"))
-            {
+        public string TestFormat(string format) {
```
Comment (microalps) [inline src/DotLiquid.Tests/Util/StrFTimeTests.cs:41]: nit: daviburg would complain about brace on same line

**Review by microalps — APPROVED** [review]

File: src/DotLiquid.Tests/Util/StrFTimeTests.cs [hunk 2]
```diff
@@ -39,12 +38,30 @@ public class StrFTimeTests
...
+
+        [Test]
+        public void TestEpoch() {
+            using (CultureHelper.SetCulture("en-GB")) {
                 Assert.That(CultureInfo.CurrentCulture, Is.EqualTo(new CultureInfo("en-GB")));
-                return new DateTime(2012, 1, 8, 14, 32, 14).ToStrFTime(format);
+                var date = new DateTime(2012, 1, 8, 14, 32, 14);
+                var localResult = date.ToStrFTime("%s");
+                Assert.AreEqual("1326033134", localResult);
+                var utcResult = new DateTimeOffset(2012, 1, 8, 14, 32, 14, TimeSpan.FromHours(0)).ToStrFTime("%s");
```
Comment (microalps) [inline src/DotLiquid.Tests/Util/StrFTimeTests.cs:61]: nit: I should read my code, could just pass date instead of 6 params.

Comment (mhornbacher) [reply src/DotLiquid.Tests/Util/StrFTimeTests.cs:61]: updating now

Comment (microalps) [reply src/DotLiquid.Tests/Util/StrFTimeTests.cs]: Thanks for updating, I was not planning on waiting for those changes just waiting for the build. We should have a nuget build shortly. Thanks again for your contribution and patience!

Comment (mhornbacher) [reply src/DotLiquid.Tests/Util/StrFTimeTests.cs]: Of course! Thank you for merging it in.

We have our own fork of the project for some internal macro logic but this seemed like something that should go upstream ;)

**Pull request merged**